fastlane/metadata/en-US/app_store_screenshot_1.txt

```diff
@@ -0,0 +1,2 @@
+A simple
+note taking experience
```

Comment:
These keep disappearing after I run a code freeze. I need to move them to a different location 😳